packages/fxa-settings/src/components/Settings/FlowSetup2faApp/en.ftl

```diff
@@ -0,0 +1,25 @@
+## FlowSetup2faApp
+
+flow-setup-2fa-qr-heading = Connect to your authenticator app
+flow-setup-2a-qr-instruction = <strong>Step 1:</strong> Scan this QR code using any authenticator app, like Duo or Google Authenticator.
+
+# Alt text for the QR-code image shown during two-step authentication setup.
+# “setup secret key” refers to the long code you can copy instead of scanning.
+# Not to be confused with the 6-digit codes generated by the authenticator app.
+flow-setup-2a-qr-alt-text =
+  .alt = QR code for setting up two-step authentication. Scan it, or choose Can’t scan QR code? to get a setup secret key instead.
```

Comment:
```suggestion
  .alt = QR code for setting up two-step authentication. Scan it, or choose “Can’t scan QR code?” to get a setup secret key instead.
```

Not sure what our style is when referencing other strings, so this is more of a question than a suggestion: Does this need quotes?

Reply:
I'm not sure we have a precedent, but it's fine by me to add these in!